test/client_options_test.py

```diff
@@ -340,6 +344,35 @@ def test_transfer_options_disabled(config):
     assert not hasattr(args, 'buffer_size')
 
 
+@pytest.mark.parametrize("format", [
+    "human",
+])
+def test_output_option(config, format):
+    parser = _options.Parser()
+    parser.add_sub_command("test", "help", lambda x: None)
+
+    args = parser.parse([
+        "test",
+        "-c", "all",
+        "--output", format
+    ])
+    assert args.output == format
```

Comment:
Why call the parameter `format` when we call the option `output`?

Reply:
Well, the command option represents how we format the output. But yes, I'll change it.